resip/stack/GenericPidfContents.hxx

```diff
@@ -71,6 +71,7 @@ public:
       EncodeStream& encode(EncodeStream& str, Data indent) const;
    };
    const NodeList& getRootNodes() const { checkParsed(); return mRootNodes; }
+   void setRootNodes(const NodeList& nodeList);
```

Comment:
If I put this change in another pull request, then the pull request #52 will not pass Travis CI build as it uses the setRootNodes() method in resip/recon/UserAgent.cxx. It is ok to remove this from here and let the Travis build fail for now?